yt/frontends/cholla/data_structures.py

```diff
@@ -157,6 +157,10 @@ def _is_valid(cls, filename, *args, **kwargs):
         except AttributeError:
             return False
         else:
-            return "bounds" in attrs and "domain" in attrs
+            return (
+                "bounds" in attrs
+                and "domain" in attrs
+                and attrs.get("data_type", None) != "yt_light_ray"
```

Comment:
None is already the default if unspecified, I don't see the value of making it explicit here. But I'm fine with the patch as is


```suggestion
                and attrs.get("data_type") != "yt_light_ray"
```